server/adaptors/integrations/__data__/repository/k8s/data/sample.json

```diff
@@ -0,0 +1,4835 @@
+[
+  {
+    "kubernetes": {
+      "node": {
+        "_domain": {
```

Comment:
Domain isn't present at all in the schema -- looks like metadata but might be worth trimming to just have a smaller storage footprint

Reply:
this info was collected by the agent so I cant remove it completely - though is in not present in either ECS nor OTEL schema...
Probably related to the specific agent collector additional collected info which is not part of the standard schema